morebits.js

```diff
@@ -1414,6 +1414,10 @@ Morebits.date = function() {
 		args[0] = args[0].replace(/(\d\d:\d\d),/, '$1').replace(/\(UTC\)/, 'UTC');
 		// Safari is particular about timezone offsets, so this is intentionally specific
 		args[0] = args[0].replace(/(\d\d:\d\d) (\d{1,2}) ([A-Z][a-z]+) (\d{4}) UTC$/, function(match, time, date, monthname, year) {
+			// zero-pad date
+			if (date < 10) {
```

Comment:
I did this to match `pad` below, but it's moderately wrong, and will break on correctly-padded dates (e.g. `07`); fixed in c0f5591.